Bottom: ced6013f2051be904fa2defd95b2ae6eef082718
Top:    50d41a996791ff0435446597f5ca34385b1ee5e9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 13:10:34 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
